.buildkite/scripts/upload-wheels.sh

```diff
@@ -74,12 +74,12 @@ fi
 aws s3 cp "$wheel" "s3://vllm-wheels/nightly/"
 aws s3 cp "$normal_wheel" "s3://vllm-wheels/nightly/"
 
-if [[ $normal_wheel == *"cu118"* ]]; then
-    # if $normal_wheel matches cu118, do not upload the index.html
-    echo "Skipping index files for cu118 wheels"
-elif [[ $normal_wheel == *"cu126"* ]]; then
+if [[ $normal_wheel == *"cu126"* ]]; then
     # if $normal_wheel matches cu126, do not upload the index.html
     echo "Skipping index files for cu126 wheels"
+elif [[ $normal_wheel == *"cu129"* ]]; then
+    # if $normal_wheel matches cu129, do not upload the index.html
+    echo "Skipping index files for cu129 wheels"
```

Comment:
we should upload cu129 wheels, and there's a requirement that both wheels for x86 and arm64 should follow the same wheel name template, so we should upload cu129 wheels for both x86 and arm64 🥺

Reply:
Got it, so cu129 for both x86 and aarch64 wheels.  Let me update the PR